app/stream/index/scoreboard/controller.js

```diff
@@ -109,6 +111,14 @@ export default Controller.extend({
       this.cloneElement = $(clone);
       this.cloneElement.on('ondrag', run.bind(this, this.onCloneDrag));
       return false;
+    },
+
+    goToStreamEditScoreboard() {
+      console.log('Goto stream edit scoreboard');
+    },
```

Comment:
This is fine for now although I think we should aim for doing edits on the same screen - so tapping 'Edit' just enables dragging etc